Form: describe Smart Paste API (#8011)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxForm/5 Item Types/SimpleItem/aiOptions/aiOptions.md

---
---
##### shortDescription

> Specifies options for AI service when you implement Smart Paste.

Reviewer: Technically, our customers do not "implement" Smart Paste. Part of that functionality is implemented by us (DevExpress). The part that parses some text into individual fields is implemented by the AI Service. 

```suggestion
Specifies options for an AI service that parses Smart Paste data.
```

Our customers can "activate" Smart Paste.

---

> Use these options to mark an item disabled for AI processing or give a custom instruction.

Reviewer: ```suggestion
Use options available in this class to customize how an AI Service processes data. You can apply the following changes: 

- Mark an field as disabled for AI processing.
- Specify custom parsing instructions for a field.
```
